README.md

```diff
@@ -1,2 +1,25 @@
 # Ruby-Tic-Tac-Toe
 Tic Tac Toe Game using OOP with ruby
+## Classical Tic-Tac-Toe in the terminal
+we have a 3X3 grid, with column 1-2-3 and rows a-b-c to choose which bracket we write the row a and the column 1
+so == a1 
+## what we had to do
+Think about how you would set up the different elements within the game… What should be a class? Instance variable? Method? A few minutes of thought can save you from wasting an hour of coding.
+
+Build your game, taking care to not share information between classes any more than you have to.
+
+Post your solution below, then check out the example solution provided.
+
+## OOP
+we used classes and methods to write and make the game work 
+
+
+# intructions 
+1: write: ruby main.rb on the terminal
+2: write the name of the 2 players
+3: select which cell with the letter of the row and the number of the column
+4: enjoy!
```

Comment:
- [ ] some rules of the game should be added, how to play the game, what is the strategy, etc, have in mind someone who has never heard about tic tac toe before